### PR TITLE
INT-2806 Create mapped relationships for subscription/location

### PR DESCRIPTION
**Pull request by eXtremeX**

### CODE REVIEW

File: src/steps/resource-manager/network/index.test.ts [hunk 1]
```diff
@@ -1714,23 +1714,26 @@ describe('rm-network-location-watcher-relationships', () => {
...
-      ).toMatchDirectRelationshipSchema({
-        schema: {
-          properties: {
-            _type: {
-              const: NetworkRelationships.LOCATION_HAS_NETWORK_WATCHER._type,
-            },
-          },
-        },
-      });
+        mappedRelationships.every(
```
Comment (mdaum) [inline src/steps/resource-manager/network/index.test.ts:1731]: note that you can use `expect(mappedRelationships).toTargetEntities` to verify your mapped relationships. [Example](https://github.com/JupiterOne/graph-veracode-v2/blob/main/src/steps/findings/index.test.ts#L63)

File: src/steps/resource-manager/subscriptions/index.test.ts [hunk 2]
```diff
@@ -136,32 +132,22 @@ describe('rm-subscription-locations', () => {
...
-    expect(subscriptionLocationRelationships).toMatchDirectRelationshipSchema({
-      schema: {
-        properties: {
-          _type: { const: relationships.SUBSCRIPTION_USES_LOCATION._type },
-        },
-      },
-    });
+    expect(subscriptionLocationMappedRelationships.length).toBeGreaterThan(0);
+    expect(
+      subscriptionLocationMappedRelationships.every(
```
Comment (mdaum) [inline src/steps/resource-manager/subscriptions/index.test.ts:140]: note that you can use `expect(subscriptionLocationMappedRelationships).toTargetEntities` to verify your mapped relationships. [Example](https://github.com/JupiterOne/graph-veracode-v2/blob/main/src/steps/findings/index.test.ts#L63)

**Review by mdaum — APPROVED** [review]
small suggestion in testing, but looks good otherwise

File: src/steps/resource-manager/network/index.test.ts (hunk 1 above)
Comment (eXtremeX) [reply src/steps/resource-manager/network/index.test.ts]: Thanks for the suggestion @mdaum, have updated the code with something similar.

**Review by neurovelho — APPROVED** [review]

**Pull request merged**